lib/phase/functional.js

```diff
@@ -223,6 +223,17 @@ module.exports = async (flattenedDoc, buildClusterFactory, triggerFactory, pipel
 
         // Workflow graph
         doc.workflowGraph = await generateWorkflowGraph(doc, triggerFactory, pipelineId);
+
+        doc.workflowGraph.nodes.forEach(node => {
```

Comment:
is this needed because it get's saved with without `~` ?

Reply:
Yeah this being getting saved without `~` was causing some problems. I needed this trigger to be the same as `~pr` and `~commit`.